helm/prometheus-rules/templates/alerting-rules/teleport.rules.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: prometheus.rules
```

This name should be unique.
```suggestion
  name: teleport.rules
```